sonar-docs/analysis/scan/sonarscanner-for-msbuild.md

```diff
@@ -65,28 +65,38 @@ SonarScanner.MSBuild.exe end
 ```
 Note: On Mac OS or Linux, you can also use `mono <path to SonarScanner.MSBuild.exe>`.
 
+If you are targeting a SonarCloud project, will have to add both the organization and a login for authentication:
+```
+SonarScanner.MSBuild.exe begin /k:"project-key" /d:sonar.organization="<organization>" /d:sonar.login="<token>"
+MSBuild.exe <path to solution.sln> /t:Rebuild
+SonarScanner.MSBuild.exe end /d:sonar.login="<token>"
+```
+
 The second version is based on .NET Core which has a very similar usage:
 ```
 dotnet <path to SonarScanner.MSBuild.dll> begin /k:"project-key"
 dotnet build <path to solution.sln>
 dotnet <path to SonarScanner.MSBuild.dll> end
 ```
 The .NET Core version can also be used as a .NET Core Global Tool.
-After installing the Scanner as a global tool as described above it can be invoked as follows: 
+After installing the Scanner as a global tool as described above it can be invoked as follows:
 ```
+dotnet tool install --global dotnet-sonarscanner
 dotnet sonarscanner begin /k:"project-key"
 dotnet build <path to solution.sln>
 dotnet sonarscanner end
 ```
 
+Same as above, if you are targetting a SonarCloud project, will have to add both the organization and a login for authentication.
```

Comment:
SonarCloud-specific, as above.

Reply:
I removed this whole paragraph and put SonarCloud-specific tags above instead